src/editor/CustomEditor.ts

```diff
@@ -55,6 +52,10 @@ const createCustomEditor = () => {
     switch (e.type) {
       case InlineElementType.InlineLatex:
         return true;
+      case BlockElementType.Image:
+        return true;
+      case BlockElementType.VoidSpacer:
+        return true;
```

Comment:
This condition would probably be better expressed as returning a boolean expression.

Reply:
I wanted to leave it open for adding more void blocks in the future, in which case a boolean would get unwieldy.

Reply:
Would it be? I was under the impression that Prettier would make long expressions multiline, which would help with the readability.

Reply:
Maybe readability is the wrong word, I think it would be clearer to use a switch on block types than parsing a long multi-line conditional.

I like the fall-through idea, I'll implement that.

Reply:
https://github.com/UWmathNEWS/slugline-web/pull/84/commits/fa74ea724f50b9f423107f681f1cb70d5c498384